modules/programs/lazygit.nix

```
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.lazygit;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+in {
+  meta.maintainers = [ maintainers.kalhauge ];
+
+  options.programs.lazygit = {
+    enable = mkEnableOption "simple terminal UI for git commands";
+
+    settings = mkOption {
+      type = yamlFormat.type;
+      default = { };
+      defaultText = literalExample "{ }";
+      example = literalExample ''
+        {
+          gui.theme = {
+            lightTheme = true;
+            activeBorderColor = [ "blue" "bold" ];
+            inactiveBorderColor = [ "black" ];
+            selectedLineBgColor = [ "default" ];
+          };
+        }'';
+      description = ''
+        Configuration written to
+        <filename>~/.config/lazygit/config.yml</filename> on linux 
+        or <filename>~/Library/Application Support/lazygit/config.yml</filename> on darwin. See
```

Very nitpicky. Feel free to ignore:

```suggestion
        Configuration written to
        <filename>~/.config/lazygit/config.yml</filename> on Linux
        or <filename>~/Library/Application Support/lazygit/config.yml</filename> on Darwin. See
```